src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchConfiguration.java

```diff
@@ -106,7 +108,10 @@ public void setElasticsearchWriteAccess(ElasticSearchWriteAccess elasticsearchWr
         this.elasticsearchWriteAccess = elasticsearchWriteAccess;
     }
 
-    protected Object readResolve() {
+    public Object readResolve() {
+        if (splitMessagesLongerThan <= 0) {
+            splitMessagesLongerThan = 2000;
```

Comment:
Why 2000? Should this be moved to a constant variable?

Reply:
> Why 2000?

I also don't understand why 2000 (which is less than 2KB) was chosen as a default value.

In the former case if the `message.length()` [was more than two times the bufferCapacity](https://github.com/SAP/elasticsearch-logs-plugin/blob/master/src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/write/fluentd/FluentdWriter.java#L177) then the message was going to be splitted. [As the default `bufferCapacity` was 1048576 (1024 * 1024 = 1MB)](https://github.com/SAP/elasticsearch-logs-plugin/blob/master/src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/write/fluentd/FluentdWriter.java#L48) hence, the message must have been more than 2MB in size so that split happens.

I made a constant out of 2000 here https://github.com/SAP/elasticsearch-logs-plugin/pull/87/commits/92a2973f4dac4491e64b517718624b2c9566fb03